Eventy/src/main/java/org/example/eventy/solutions/controllers/ProductController.java

```diff
@@ -31,26 +33,28 @@ public class ProductController {
        also SolutionCardDTO == ProductCardDTO == ServiceCardDTO (only a few of fields will be null) */
     // GET "/api/products"
     // ***NOTE: this should be named getProducts PROBABLY, but we already have one method below named like that
-    @GetMapping(value = "/cards", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<Collection<SolutionCardDTO>> getProductCards(Pageable pageable) {
-        ArrayList<Solution> productModels = productService.getProducts(pageable);
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<PagedResponse<SolutionCardDTO>> getProducts(Pageable pageable) {
```

Comment:
Do we need a separate method for getting products/services or is it enough to have one getSoultions method with type filter?

Reply:
Idk honestly, if someone needs it, it's there